src/components/AboutDialog.vue

```diff
@@ -1,49 +1,62 @@
 <template>
   <v-dialog
-    class="ws-dialog"
     v-model="dialog"
+    class="ws-dialog"
     width="500"
-    >
+  >
     <template v-slot:activator="{ on }">
       <v-btn
         class="mt-n1 ml-n1"
         icon
         small
         v-on="on"
       >
-        <v-icon size="18">help</v-icon>
+        <v-icon size="18">
+          help
+        </v-icon>
       </v-btn>
     </template>
 
     <v-card>
-
       <v-card-title
         class="headline pb-0 pt-3"
         primary-title
-        >
+      >
         About Multinet
       </v-card-title>
 
       <v-card-text
         class="px-4 pt-4 pb-1"
-        >
+      >
         Multinet is a system for storing and processing <a-ext
-          href="https://vdl.sci.utah.edu/mvnv/">multivariate networks</a-ext>. Learn more and explore the code
-        at <a-ext href="https://github.com/multinet-app/multinet">GitHub</a-ext>.
+          href="https://vdl.sci.utah.edu/mvnv/"
+        >
+          multivariate networks
+        </a-ext>. Learn more and explore the code
```

Comment:
Looks like formatting the `a-ext` tags like this adds a space to the end of the links

Reply:
See c22cfa9425286eea7299afcb13bfc43ee53a970e.